README.pt-br.md

```diff
@@ -1295,6 +1296,31 @@ get '/:value' do
   session['value'] = params['value']
 end
 ```
+#### Chave de segurança da sessão
+
+Para melhorar a segurança, os dados da sessão no cookie são assinados com a chave de segurança da sessão
+usando `HMAC-SHA1`. Essa chave deve ser preferencialmente um
+valor randômico, criptograficamente seguro, com tamanho apropriado para
+`HMAC-SHA1`, maior ou igual a 64 bytes (512 bits, 128
+caracteres hexadecimais). Você seria orientado a não usar uma chave randômica menor que 32
```

Comment:
I would change it to be "Aconselha-se que não se use [...]"

Reply:
I agree!